src/main/java/net/minecraftforge/client/event/EntityViewRenderEvent.java

```diff
@@ -43,6 +46,7 @@ public double getPartialTicks()
         return partialTicks;
     }
 
+    //TODO: remove in 1.19
```

Comment:
This should be a `@Deprecated(forRemoval = true, since = "1.18.2")` annotation and a javadoc with a `@deprecated` tag explaining that it will be removed in 1.19 and why it will be removed.